src/Pretzel.Tests/Extensibility/Extensions/WebSequenceDiagramsTests.cs

```diff
@@ -58,5 +58,26 @@ public void Multiple_blocks_are_converted_to_multiple_diagrams()
             var markdown = transform.Transform(input);
             Assert.Equal(expected.RemoveWhiteSpace(), markdown.RemoveWhiteSpace());
         }
+
+        [Fact]
+        public void Single_block_is_converted_to_diagram_style_mscgen()
+        {
+            const string input = @"<p>hello</p>
+<pre><code>@@sequence mscgen
+a->b: foo
+b->a: bar
+</code></pre>
+<p>world</p>";
+
+            const string expected = @"<p>hello</p>
+<div class=""wsd"" wsd_style=""mscgen""><pre>a->b: foo
+b->a: bar
+</pre></div>
+<p>world</p>
+<script type=""text/javascript"" src=""http://www.websequencediagrams.com/service.js""></script>";
+
+            var markdown = transform.Transform(input);
+            Assert.Equal(expected.RemoveWhiteSpace(), markdown.RemoveWhiteSpace());
```

Comment:
ApprovalTests is awesome for this sort of test. https://github.com/approvals/ApprovalTests.Net